<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>880</width>
    <height>607</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>220</y>
      <width>111</width>
      <height>86</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="5" column="0">
      <widget class="QTimeEdit" name="time_out_edit"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="time_in_label">
       <property name="text">
        <string>Time In</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="time_out_label">
       <property name="text">
        <string>Time Out</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QTimeEdit" name="time_in_edit"/>
     </item>
    </layout>
   </widget>
   <widget class="QTableView" name="times_table">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>10</y>
      <width>661</width>
      <height>391</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="record_button">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>340</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Record</string>
    </property>
   </widget>
   <widget class="QComboBox" name="employee_combo">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>130</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Example Name</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Example Name 2</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="employee_label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>100</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Select Employee: </string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
